pages/dao/[symbol]/proposal/new.tsx

```diff
@@ -181,24 +195,34 @@ const getDefaultInstructionProps = (
 
 const New = () => {
   const router = useRouter()
-  const { handleCreateProposal } = useCreateProposal()
+  const { handleCreateProposal, proposeMultiChoice } = useCreateProposal()
   const { fmtUrlWithCluster } = useQueryContext()
   const realm = useRealmQuery().data?.result
 
-  const { symbol, realmInfo, canChooseWhoVote } = useRealm()
+  const { symbol, realmInfo, canChooseWhoVote} = useRealm()
   const { availableInstructions } = useGovernanceAssets()
   const [voteByCouncil, setVoteByCouncil] = useState(false)
   const [form, setForm] = useState({
     title: typeof router.query['t'] === 'string' ? router.query['t'] : '',
     description: '',
   })
+  const [multiChoiceForm, setMultiChoiceForm] = useState<{
+    governedAccount: AssetAccount | undefined
```

Comment:
don't put queried data in application state (I realize it's commonly done in this codebase). 

I thought I'd deprecated `AssetAccount`, maybe you are behind main.

Anyway just use the pubkey of the governance. LMK if you have questions about this, I know the code is confusing.